### PR TITLE
Develop 0.3.1

### PR DESCRIPTION
**Pull request by VuksanFilip**

### CODE REVIEW

File: HospitalSystem.ConsoleUI/Secretary/EquipUI.cs [hunk 1]
```diff
@@ -212,18 +213,19 @@ public void EquipmentTable(List<EquipmentBatch> equipments, string header)
...
-            System.Console.WriteLine(String.Format("| {0,-8} | {1,-9} | {2, -5} |", equipment.RoomLocation, equipment.Name, equipment.Count));
+            List<Room> stocks = _hospital.RoomService.GetStocks().ToList();
+            if(!stocks.Any(stock => stock.Location == equipment.RoomLocation)){
+                if (equipment.Count == 0)
+                {
+                System.Console.WriteLine(String.Format("|[{0,-8}]|[{1,-9}]|[{2, -5}]|", equipment.RoomLocation, equipment.Name, equipment.Count));
+                }
+                else
+                {
+                System.Console.WriteLine(String.Format("| {0,-8} | {1,-9} | {2, -5} |", equipment.RoomLocation, equipment.Name, equipment.Count));
```
Comment (swopl) [inline HospitalSystem.ConsoleUI/Secretary/EquipUI.cs:224]: Why are these two nearly identical lines in different blocks? Do they even need to be different?

**Review by swopl — CHANGES_REQUESTED** [review]

File: HospitalSystem.Core/Rooms/IRoomRepository.cs [hunk 2]
```diff
@@ -17,4 +17,6 @@ public interface IRoomRepository
     public void Upsert(Room room, Expression<Func<Room, bool>> filter);
 
     public bool DoesExist(string location);
+
+    public IQueryable<Room> GetStocks();
```
[the remark this answers is not in the document]
Comment (VuksanFilip) [reply HospitalSystem.Core/Rooms/IRoomRepository.cs:21]: so those are Stocks

[the remark this answers is not in the document]
Comment (swopl) [reply HospitalSystem.Core/Rooms/IRoomRepository.cs:21]: No, they are Stock Rooms, not stocks. Name this function either GetAllStock or GetStock. The plural of Stock Room is Stock Rooms

File: HospitalSystem.ConsoleUI/Secretary/EquipUI.cs (hunk 1 above)
Comment (swopl) [reply HospitalSystem.ConsoleUI/Secretary/EquipUI.cs:224]: How are they emphasized?

File: HospitalSystem.Core/Rooms/IRoomRepository.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (VuksanFilip) [reply HospitalSystem.Core/Rooms/IRoomRepository.cs:21]: Just let it be like that, those are clearly Stocks, why should i put a Room inside a function, it is clearly t a Stock is a Room!

File: HospitalSystem.ConsoleUI/Secretary/EquipUI.cs (hunk 1 above)
Comment (VuksanFilip) [reply HospitalSystem.ConsoleUI/Secretary/EquipUI.cs:224]: With [] brackets

Comment (VuksanFilip) [reply HospitalSystem.ConsoleUI/Secretary/EquipUI.cs:224]: professor: "Specijalno je označena roba koje nema u prostorijama da bi 
je sekretar lakše uočio. "

**Pull request merged**